model/config.go

```diff
@@ -513,11 +513,6 @@ func (o *Config) SetDefaults() {
 		o.FileSettings.AmazonS3Endpoint = "s3.amazonaws.com"
 	}
 
-	if o.FileSettings.AmazonS3Region == "" {
```

Comment:
Why are we removing the default region? I don't think there's a case where we want the user to be setting this to be empty